fake_ui.py

```diff
@@ -0,0 +1,16 @@
+from User import User
+
+# these would be user inputs
```

Comment:
Is this needed? I don't think UI is relevant to the pull request.

Reply:
I made a file to show how the code should've been ran. For lack of a better name, I named it fake_user. I'll keep it locally though.